tensorflow/contrib/rnn/python/kernel_tests/core_rnn_cell_test.py

```diff
@@ -663,6 +663,12 @@ def _testDropoutWrapper(self, batch_size=None, time_steps=None,
         self.assertEqual(res[1].h.shape, (batch_size, 3))
         return res
 
+  def testProperties(self):
+    cell = rnn_cell_impl.BasicRNNCell(4)
+    wrapper = rnn_cell_impl.DropoutWrapper(cell)
+    # Github issue: 15810
```

Comment:
Add a test when some variables have been added.

Reply:
Remove the GitHub comment

Reply:
New test has been added.